### DISCUSSION TITLE
Auto-Collapse functionality for Array items

### DISCUSSION THREAD

**Discussion by PatrikKozak** [payloadcms/payload#4898] [Feature Requests & Ideas]
Currently, multiple array items can be expanded (uncollapsed) simultaneously within an array field.

`Feature Request`: Add option where only one array item can be expanded at a time. When a user expands one item, any previously expanded item would automatically collapse.

**Comment by tylandavis** [comment]
This worth be a nice item to have in Payload user settings, with dark/light mode and language. I could see some users wanting this and some not, so having it outside of the config would be ideal imo.

bonus: I think it'd be worth exploring a dropdown for the user avatar that lets users adjust these kind of settings without needing to navigate away from the current document.

**Reply by tak-amboss** [reply]
That's interesting. We have a different use-case: We customize (re-style) the array component heavily in our implementation so that it doesn't make sense from a UI perspective to have multiple items uncollapsed for certain fields. In this case it's rather something we'd want to configure as a general behavior on a field level.
